### PR TITLE
Upgrade to Airflow 2.0.0

### PR DESCRIPTION
**Pull request by danielvdende**
This PR:
- Updates Whirl to use Airflow 2.0.0 by default. 
- Fixes imports for Airflow 2.0.0 where necessary. 
- Updates creation of Airflow connections with CLI to new CLI version.
- A user-password combination is now mandatory (otherwise Airflow's web UI won't let you in). To make life easy, this is set to `admin-admin`

### CODE REVIEW

File: envs/dbt-databricks-example/.whirl.env [hunk 1]
```diff
@@ -0,0 +1,24 @@
+AIRFLOW_VERSION=1.10.14
```
Comment (BasPH) [inline envs/dbt-databricks-example/.whirl.env:1]:
```suggestion
AIRFLOW_VERSION=2.0.0
```

File: envs/dbt-databricks-example/.whirl.env [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+POSTGRES_PASSWORD=p@ssw0rd
+POSTGRES_USER=postgres
+POSTGRES_DB=postgresdb
+
+# Spark variables
+SPARK_VERSION=3.0
+
+# Airflow env vars
+AIRFLOW__CORE__EXPOSE_CONFIG=True
+AIRFLOW__WEBSERVER__EXPOSE_CONFIG=True
```
Comment (BasPH) [inline envs/dbt-databricks-example/.whirl.env:22]:
```suggestion
AIRFLOW__WEBSERVER__EXPOSE_CONFIG=True
```

File: envs/postgres-s3-external-spark/whirl.setup.d/03_add_spark_config.sh [hunk 3]
```diff
@@ -7,13 +7,12 @@ sudo apt-get update && sudo apt-get install -y openjdk-11-jre
 export JAVA_HOME=/usr/lib/jvm/java-11-openjdk-amd64
 
 echo "============================"
-echo "== Configure Spark config =="
+echo "== Configure Spark configXXX =="
```
Comment (BasPH) [inline envs/postgres-s3-external-spark/whirl.setup.d/03_add_spark_config.sh:10]: typo?

**Comment by danielvdende** [comment]
@krisgeus @BasPH think this is ready for review now. Let me know what you think 🙂

File: examples/spark-s3-to-postgres/whirl.setup.d/00_install_airflow_providers.sh [hunk 4]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+echo "========================================"
+echo "== Install required airflow providers =="
+echo "========================================"
+
+pip install apache-airflow-providers-apache-spark
+aws s3api put-object --bucket ${DEMO_BUCKET} --key input/data/demo/spark/$(date "+%Y%m%d")/data.json --body /mock-data/input.json
```
Comment (krisgeus) [inline examples/spark-s3-to-postgres/whirl.setup.d/00_install_airflow_providers.sh:8]: Isn;t this already present in 01_add_mockdata?

Comment (danielvdende) [reply examples/spark-s3-to-postgres/whirl.setup.d/00_install_airflow_providers.sh]: good catch 👍 Some fat-finger copy-pasting went on there 😉

**Pull request merged**